.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+
+# Controls when the workflow will run
+on:
```

Comment:
For our CI, we have everywhere the same appropriate trigger configuration. Please copy:
https://github.com/openzim/libzim/blob/1bcb213845d4cdda5525b095ebe5fe9b3d0e93f8/.github/workflows/ci.yml#L3

Reply:
updated